climate_categories/data/RCMIP.yaml

```diff
@@ -0,0 +1,439 @@
+name: RCMIP
+title: Emissions categories from the Reduced Complexity Model Intercomparison Project
+  (RCMIP)
+comment: "AFOLU in the SSPDb is AFOLU minus any agriculture\nrelated fossil fuel based\
+  \ emissions hence is not the same as the\nWG3 AFOLU definition. Rather AFOLU in\
+  \ the SSPDb is AFOLU as expected by\nMAGICC (i.e. exluding agriculture related fossil\
+  \ fuel use), hence\nwe call it MAGICC AFOLU."
+references: 'Nicholls, Z. R. J., Meinshausen, M., Lewis, J., Gieseke, R., Dommenget,
+  D., Dorheim, K., Fan, C.-S., Fuglestvedt, J. S., Gasser, T., Golüke, U., Goodwin,
+  P., Hartin, C., Hope, A. P., Kriegler, E., Leach, N. J., Marchegiani, D., McBride,
+  L. A., Quilcaille, Y., Rogelj, J., Salawitch, R. J., Samset, B. H., Sandstad, M.,
+  Shiklomanov, A. N., Skeie, R. B., Smith, C. J., Smith, S., Tanaka, K., Tsutsui,
+  J., and Xie, Z.: Reduced Complexity Model Intercomparison Project Phase 1: introduction
+  and evaluation of global-mean temperature response, Geosci. Model Dev., 13, 5175–5190,
+  https://doi.org/10.5194/gmd-13-5175-2020, 2020.'
+institution: RCMIP
+hierarchical: true
+last_update: '2020-09-21'
+version: v5.1.0
+total_sum: true
+canonical_top_level_category: Emissions
+categories:
+  Emissions:
+    title: RCMIP Emissions
+    children:
+    - - Emissions|BC
+      - Emissions|CH4
+      - Emissions|CO
+      - Emissions|CO2
+      - Emissions|F-Gases
+      - Emissions|Montreal Gases
+      - Emissions|N2O
+      - Emissions|NH3
+      - Emissions|NOx
+      - Emissions|OC
+      - Emissions|Sulfur
+      - Emissions|VOC
+  Emissions|BC:
+    title: Black carbon emissions
+    comment: total black carbon emissions
+    children:
+    - - Emissions|BC|MAGICC AFOLU
+      - Emissions|BC|MAGICC Fossil and Industrial
+      - Emissions|BC|Other
+  Emissions|BC|MAGICC AFOLU:
+    title: Black carbon AFOLU emissions
+    comment: black carbon emissions from agriculture, forestry and other land use
+      (IPCC category 3), excluding any fossil-fuel based emissions in the Agricultural
+      sector (hence not identical to WG3 AFOLU)
+  Emissions|BC|MAGICC Fossil and Industrial:
+    title: Black carbon fossil and industrial emissions
+    comment: black carbon emissions from energy use on supply and demand side (IPCC
+      category 1A, 1B), industrial processes (IPCC category 2), waste (IPCC category
+      4) and other (IPCC category 5)
+  Emissions|BC|Other:
+    title: Black carbon emissions from other sources
+    comment: black carbon emissions from other sources (please provide a definition
+      of other sources in this category in the 'comments' tab)
```

Comment:
I think in this context, the "'comments' tab" doesn't make sense any more. Maybe just `please provide a definition`?

Reply:
Good catch, i'll fix that!

Reply:
Oh, you were faster :-)
Thx!

Reply:
I'd remove the parentheses comment completely, i think it makes only sense for submitters in the RCMIP context, not for people trying to look at the data/visualizing it.

Reply:
I dropped the comments tab.

Reply:
Hmm, for me, it was helpful when looking at CRF data that the people who invented the terminology actually wanted users of the terminology to specify what they meant by "other". Especially because not everyone actually does so.

Reply:
Yeah, good point. I'm not sure how it would actually be used in RCMIP. Keeping it like you did is also fine!